examples/axisartist/demo_floating_axes.py

```diff
@@ -1,3 +1,10 @@
+"""
+Demo of the scatter function and bar function with changing the shape of the plot.
```

Comment:
I'd make the title shorter, and about the features showcases here: something as simple as "Floating axes" would work well here.